### ISSUE TITLE
Specify path for nyc module

### ISSUE THREAD

**Issue by andrewbusch7** [maty21/mocha-sidebar#206]
### Description
I'm currently getting the following error as a popup in vscode when I try enable the extension:

```
side-bar: cant find nyc locally and there is no setting for other path,
Error: get-installed-path: module not found "nyc" in path
/Users/me/Repositories/thisRepo/src/node_modules/nyc
```

I have nyc at `/Users/me/Repositories/thisRepo/src/tests/node_modules/nyc`

I have already set `mocha.path` to point to this `node_modules` directory yet I still get the error. It would be nice to either be able to set the path specifically for `nyc`, or have one place to set all node modules.

#### Settings file
```
{
	"mocha.options": { "compilers": "ts-node/register" },
	"mocha.requires": ["ts-node/register"],
	"mocha.files.glob": "test/*.test.ts",
	"mocha.path": "tests/node_modules/mocha"
}
```

### Versions

- VScode: v1.38.1
- Mocha SideBar: v0.22.2
- Node Version: v10.15.0

### OS version

- [ ] Linux 
- [X] Mac
- [ ] Windows

**Comment by sanket-dev** [comment]
Yes, this is much-needed feature. I am not able to use the mocha sidebar plugin because of this as my node_modules folder is not located at the root of the project so the plugin is not able to find nyc.

**Comment by RyanJMcGowan** [comment]
This may seem obvious, but be sure your working directory is the root level of the node_modules folder.

**Comment by evil-shrike** [comment]
Any news?
I have a VSCode workspace with 2 projects (for backend and frontend), it seems that it's impossible to use Moch sidebar in such a configuration :(

**Comment by chiorji** [comment]
> Any news?
> I have a VSCode workspace with 2 projects (for backend and frontend), it seems that it's impossible to use Moch sidebar in such a configuration :(

Having same problem, no fix yet?

**Comment by melibe23** [comment]
Same issue, no workarounds?

**Comment by natalygoloborodko** [comment]
Any news?

**Comment by rugk** [comment]
I guess this is a duplicate of https://github.com/maty21/mocha-sidebar/issues/181, you just want to set your nyc path.
